### PR TITLE
bluebinder: support reading the bluetooth address from persist.vendor.service.bdroid.bdaddr.

### PR DESCRIPTION
**Pull request by mkosola**

### CODE REVIEW

**Comment by mlehtima** [comment]
We could consider moving all bluetooth address handling to the new post script.

File: bluebinder_post.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+if [ "$(getprop persist.vendor.service.bdroid.bdaddr)" != "" ]; then
+    mkdir -p /var/lib/bluetooth
+    echo $(getprop persist.vendor.service.bdroid.bdaddr |awk -F: '{do printf "%s"(NF>1?FS:RS),$NF;while(--NF)}') > /var/lib/bluetooth/board-address
+    chown root:root /var/lib/bluetooth/board-address
+    chmod 644 /var/lib/bluetooth/board-address
+    exit 0
+fi
+
+exit 0
```
Comment (krnlyng) [inline bluebinder_post.sh]: I think we should check here if /var/lib/bluetooth/board-address was generated, and if it was not then exit 1 instead

**Comment by mkosola** [comment]
> We could consider moving all bluetooth address handling to the new post script.

Updated PR and moved whole handling to post script

**Comment by sledges** [comment]
Delete this commit please (it's been rendered inexistent anyway:): "bluebinder: fallback to alternative method to obtain BT address"

**Comment by mkosola** [comment]
> Delete this commit please (it's been rendered inexistent anyway:): "bluebinder: fallback to alternative method to obtain BT address"

Done

**Pull request merged**